meteor/override-fastclick.js

```diff
@@ -0,0 +1,6 @@
+// disable fastclick package in Meteor if included
```

Comment:
If there's a file that accommodates a package manager, cool. So package.js is ok. I'd like to avoid functional code that gets added to the ionic.bundle.min.js whose only purpose is to accommodate other libraries. Is it possible to add that bit of code to another repo and include it via package.js? I know it seems silly, but it's a precedent that could bite us later if we don't do things right.